skyportal/handlers/api/user_obj_list.py

```diff
@@ -0,0 +1,358 @@
+import re
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    User,
+    Obj,
+    Listing,
+)
+
+
+class UserObjListHandler(BaseHandler):
+    @auth_or_token
+    def get(self, user_id):
```

Comment:
Since this endpoint will be hit via the front-end, do we want to just use the current user like we do for the profile handler?

Reply:
I think if anything, the `user_id` should be optional, with the default value equal to current user. 
I'm not sure how exactly to get the current user though...

Reply:
yes, good call on making it optional. if not provided, you can access the current user inside the handle with `self.current_user`